docs/getting-started/create-defend-policy-api.asciidoc

```diff
@@ -0,0 +1,819 @@
+[[create-defend-policy-api]]
+= Create an {elastic-defend} policy using API
+
+:frontmatter-description: Create and customize an {elastic-defend} policy through the API.
+:frontmatter-tags-products: [security, defend]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet-guide}/fleet-api-docs.html[{fleet} API]. You can repeat steps 2 and 3 to make more modifications to the {elastic-defend} policy.
+
+[discrete]
+[[create-agent-policy]]
+== Step 1: Create an agent policy
+
+Make the following API call to create a new agent policy where you will add your {elastic-defend} integration. Replace `<KIBANA-VERSION>` with your version of {kib}.
+
+[source,console]
+----
+curl --user <username>:<password> --request POST \
+  --url 'https://<kibana-url>:5601/api/fleet/agent_policies' \
+  -H 'Accept: */*' \
+  -H 'Accept-Language: en-US,en;q=0.9' \
+  -H 'Connection: keep-alive' \
+  -H 'Content-Type: application/json' \
+  -H 'Sec-Fetch-Dest: empty' \
+  -H 'Sec-Fetch-Mode: cors' \
+  -H 'Sec-Fetch-Site: same-origin' \
+  -H 'kbn-version: <KIBANA-VERSION>' \ <1>
+  --data-raw '{
+  "name": "My Policy Name",
+  "description": "",
+  "namespace": "default",
+  "monitoring_enabled": [
+    "logs",
+    "metrics"
+  ],
+  "inactivity_timeout": 1209600,
+  "is_protected": false
+}' \
+  --compressed
+----
+<1> `<KIBANA-VERSION>` to be replaced
+
+Make a note of the `<POLICY-ID>` you receive in the response. You will use this in step 2 to add {elastic-defend}.
+
+.Click to see example response
+[%collapsible]
+====
+[source, json]
+----
+{
+  "item": {
+    "id": "<POLICY-ID>", <1>
+    "name": "My Policy Name",
+    "description": "",
+    "namespace": "default",
+    "monitoring_enabled": [
+      "logs",
+      "metrics"
+    ],
+    "inactivity_timeout": 1209600,
+    "is_protected": false,
```

Comment:
this field you should leave in even though we deleted from the request above.  It's an optional field that will get added whether you include it or not.  This is not an important detail for the purposes of this PR, I just looking to simplify the request where I could.

Reply:
Got it, thanks for the explanation!